lib/spotlight_search/utils.rb

```diff
@@ -2,7 +2,9 @@
 
 module SpotlightSearch::Utils
   class << self
-    def serialize_csv_columns(*columns, **hashes)
+    def serialize_csv_columns(*columns)
+      hashes = columns.select{|associations| associations if associations.class.eql?(Hash)}.last || {}
+      columns = columns.select{|column| column if column.class.eql?(Symbol)}
```

Comment:
1. Not sure why you would name it as plural. `associations`
2. You do not need to return in select. Check [here](https://www.geeksforgeeks.org/ruby-array-select-function/)
3. if the columns going to contain both hash and symbol, then we can keep the variable name inside select as columnable in both places right?
4. I would suggest the second variable to be different, because you will be losing original data. This is optional, if you think its proper ignore it.